src/cli.rs

```diff
@@ -201,7 +201,7 @@ pub fn ask_user_pass(opt: &Opt) -> Result<(String, String)> {
 			))?
 		} else {
 			pass = String::from_utf8(pw_out.stdout).map(|x| {
-				x.trim_end().to_string()
+				x.lines().collect::<Vec<_>>().first().unwrap_or(&x.as_str()).to_string()
```

Comment:
```suggestion
				x.lines().next().map(|x| x.to_owned()).ok_or_else(|| anyhow!("empty pass(1) entry"))
```

Reply:
Oof yeah, that's way better, should write way more Rust code these days :D